pages/index.mdx

```diff
@@ -34,23 +36,68 @@ To check it, run `node --version{:shell}` — it should show you the version 18.
   </Tabs.Tab>
 </Tabs>
 
-### That's it! Your first contract project is written and compiled already!
-
-Go check it out by moving into the relevant directory — `cd simple-counter/contracts{:shell}`.
-
-To re-compile, use commands in the scripts section of `package.json` in the root of this newly created project.
-
-To deploy, refer to the documentation of [Blueprint](https://github.com/ton-org/blueprint) — this is a tool we've used to create and compile your first simple counter contract in Tact. In fact, Blueprint can do much more than that: including tests, customizations and more.
+### That's it! [#start-3]
+
+Your first contract project is written and compiled already! Go check it out by moving into the relevant directory — `cd simple-counter/contracts{:shell}`.
+
+It will look like this, but without the comments:
```

Comment:
I'd prefer we move this heavily commented contract to a page like https://learnxinyminutes.com/docs/tact, but on the landing page we just give a taste of what contracts in Tact look like, without explanations. Just provide look-and-feel, basically.

Reply:
Agreed, went a bit overboard so it doesn't fit in the screen :)
Moved it to https://github.com/tact-lang/awesome-tact/issues/21#issuecomment-1996771703